tests/DocumentationExamplesTest.php

```diff
@@ -40,12 +40,10 @@ public function setUp(): void
 
     public function tearDown(): void
     {
-        if ($this->hasFailed()) {
-            return;
+        if (! $this->hasFailed()) {
+            $this->dropCollection();
```

Comment:
Noted that this will likely change in https://github.com/mongodb/mongo-php-library/pull/1075 but it's fine for this PR.